app/workflow_manager_proc/tests/test_analysis_run.py

```diff
@@ -292,7 +292,13 @@ def test_get_arsc_hash(self):
             analysis=test_analysis,
             computeEnv="ENV1",
             storageEnv="ENV2",
-            status="TESTING"
+            status="TESTING",
+            libraries=[
+                arsc.Library(
+                    orcabusId="SOME9ARSC9LIBRARY9ID12345",
```

Comment:
FYI: this is not a valid ULID. (the `I` character is excluded as it could be mistaken for a `1`)
(but I guess to test the hashing that does not matter)